consensus/broadcast/metaChainMessenger.go

```diff
@@ -112,6 +117,22 @@ func (mcm *metaChainMessenger) BroadcastHeader(header data.HeaderHandler, pkByte
 	return nil
 }
 
+// BroadcastEquivalentProof will broadcast the proof for a header on the metachain common topic
+func (mcm *metaChainMessenger) BroadcastEquivalentProof(proof *block.HeaderProof, pkBytes []byte) error {
+	if check.IfNilReflect(proof) {
+		return spos.ErrNilHeader
+	}
+
+	msgProof, err := mcm.marshalizer.Marshal(proof)
+	if err != nil {
+		return err
+	}
+
+	mcm.broadcast(common.EquivalentProofsTopic, msgProof, pkBytes)
```

Comment:
is this a topic between metachain and all shards?

Reply:
fixed